tests/dat/actions/asyncError.js

```diff
@@ -0,0 +1,13 @@
+function main(params) {
+    var str = params.payload;
+
+    return whisk.invoke({
+        name: 'wc',
```

Comment:
since the point is for this to fail, change `'wc'` to an invalid action name: `'#'`.